README.md

```diff
@@ -72,13 +72,18 @@ Returns a health check.
 
 ### POST /api/notifications
 #### Input
- 
+
 - `toUserId`: Id of the user the notification is for.
 - `notificationType`: Type of the notification being saved.
 
 #### Output
+
 - `notificationId`: Id of the notification that got created.
 
+## Design Doc
+
+  link: [DESIGN.md](docs/DESIGN.md)
```

Comment:
I am not able to navigate to this design doc. Is it because it is not merged? I can do this locally.

Reply:
Looking at the [version on your branch](https://github.com/PermanentOrg/notification-service/blob/PER-8245/README.md#design-doc), the link seems to work for me. Where are you trying to view it?

Reply:
Yes I can see it. May be I was looking at the wrong place.